packages/hawtio/src/ui/page/HawtioHeader.tsx

```diff
@@ -104,9 +105,27 @@ const HawtioHeaderToolbar: React.FunctionComponent = () => {
     userItems.pop()
   }
 
+  /*
+   * Determine which plugin is currently displaying
+   * based on the path of the current location
+   */
+  const pluginFromLocation = (): Plugin | null => {
+    const path = location.pathname
+    if (!path) return null
+
+    return plugins.find(plugin => plugin.path === path) || null
```

Comment:
does this test still work if the location is under a subpath such as `/jmx/attributes`?  shouldn't it use `startsWith`?